vulnerabilities/importers/debian.py

```diff
@@ -108,6 +108,8 @@ def _parse(self, pkg_name: str, records: Mapping[str, Any]) -> List[Advisory]:
 
         for cve_id, record in records.items():
             impacted_purls, resolved_purls = set(), set()
+            if not cve_id.startswith("CVE"):
```

Comment:
```suggestion
            if not cve_id.startswith("CVE"):
                 # vulnerabilities starting with something may not be public yet
                 # see for instance https://web.archive.org/web/20201215213725/https://security-tracker.debian.org/tracker/TEMP-0000000-A2EB44
                 # TODO: this would need to be revisited though to ensure we are not missing out on anything
```

Reply:
Shouldn't that be `with something else may not be public` ?